tests/src/Context/JoinupSearchContext.php

```diff
@@ -374,6 +462,47 @@ public function checkCheckboxFacet(string $option, string $facet_type): void {
     throw new \Exception("The option '{$option}' was not found in the '{$facet_type}' facet.");
   }
 
+  /**
+   * Checks a checkbox in a facet form.
+   *
+   * @param string $option
+   *   The label of the checkbox.
+   * @param string $facet_type
+   *   The label of the facet.
+   *
+   * @throws \Exception
+   *   Thrown when the checkbox is not found.
+   *
+   * @Given I check the :option checkbox from the :facet_type facet form
+   */
+  public function checkCheckboxFacetForm(string $option, string $facet_type): void {
+    $facet = $this->findFacetFormByAlias($facet_type);
+    /** @var \Behat\Mink\Element\NodeElement[] $node_elements */
+    $node_elements = $facet->findAll('xpath', '//option');
+    foreach ($node_elements as $node_element) {
+      if ($node_element->getText() === $option) {
+        $node_element->click();
+        return;
+      }
+    }
+
+    throw new \Exception("The option '{$option}' was not found in the '{$facet_type}' facet.");
+  }
+
+  /**
+   * Facets form inputs.
+   *
+   * @param string $name
+   *   The label of the input.
```

Comment:
Also, input is not really a good word for action buttons. I mean, I had to look at the method to understand what it does.